server/jetstream_api.go

```diff
@@ -2444,11 +2444,11 @@ func (s *Server) jsLeaderServerStreamCancelMoveRequest(sub *subscription, c *cli
 
 	peers := currPeers[:cfg.Replicas]
 
-	s.Noticef("Requested cancel of move: R=%d stream %s in account %s from old peer set %+v to new peer set %+v",
+	s.Noticef("Requested cancel of move: R=%d '%s > %s' from old peer set %+v to new peer set %+v",
 		cfg.Replicas, streamName, accName, s.peerSetToNames(currPeers), s.peerSetToNames(peers))
 
-	// execute on a go routine for route or gateway, as this may block waiting from responses from other servers and
-	// therefore block all route/gateway traffic.
+	// If we are inline with client, we still may need to do a callout for stream info
+	// during this call, so place in Go routine to not block client.
```

Comment:
Again, here I don't think jsLeaderServerStreamCancelMoveRequest is invoked from apiDispatch, but directly subscribed to:
```
if cc.peerStreamCancelMove == nil {
		cc.peerStreamCancelMove, _ = s.systemSubscribe(JSApiServerStreamCancelMove, _EMPTY_, false, c, s.jsLeaderServerStreamCancelMoveRequest)
	}
```